### PR TITLE
Sign-up

### PR DESCRIPTION
**Pull request by ardeshirj**
This pull request is for issue #3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,40 @@
-Capn-Crunch
-===========
+# Capn-Crunch
```
Comment (lewisric) [inline README.md]: Thanks for the readme update

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -1,2 +1,33 @@
-<h1>Users#new</h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<% provide(:title, 'Sign up') %>
+<h1>Sign up</h1>
+
+<%= form_for(@user) do |f| %>
```
Comment (lewisric) [inline app/views/users/new.html.erb]: @ardeshirj can we get a more descriptive name other than "f"

**Comment by lewisric** [comment]
We're going to have a merge issue, if we can get another person too look at the code then we can fix the merge issues.
@ardeshirj 
Can one or two of you take a gander at the updated files
@mindaika 
@CATcommander 
@beazley2 
@sector46

File: README.md (hunk 1 above)
Comment (mindaika) [inline README.md]: Looks real nice. That's a real nice readme. Real nice.

**Pull request merged**